pkg/devspace/config/loader/validate.go

```diff
@@ -393,8 +397,8 @@ func validateDev(config *latest.Config) error {
 	if config.Dev.Sync != nil {
 		for index, sync := range config.Dev.Sync {
 			// Validate imageName and label selector
-			if len(sync.LabelSelector) == 0 && sync.ImageSelector == "" {
-				return errors.Errorf("Error in config: image selector and label selector are nil in sync config at index %d", index)
+			if sync.ContainerName != "" && len(sync.LabelSelector) == 0 && sync.ImageSelector == "" {
```

Comment:
These need to be separte checks:
1. containerName != "" && len(labelSelector) == 0
2. len(labelSelector) == 0 && imageSelector == ""